service/app/distribution_test.go

```diff
@@ -65,14 +69,18 @@ func TestDistributionResolution(t *testing.T) {
 			PackCount: 4,
 			Buckets: []Bucket{
 				{
-					CollectibleCount:      2,
 					CollectibleCollection: bucket1,
 				},
 				{
-					CollectibleCount:      2,
 					CollectibleCollection: bucket2,
 				},
 			},
+			Slots: []SlotTemplate{
+				{BucketIndexes: []int{0}},
```

Comment:
Are we now reverting back to the original design where the slots define which buckets they can be filled with?

Reply:
No, this PR is just to compare the implementations.

Reply:
Going to delete and re-create as a draft 👍